Dockerfile

```diff
@@ -51,9 +51,8 @@ RUN apt-get install -y --no-install-recommends \
 # -----------------------------------------------------------------------------
 # Install Python
 # -----------------------------------------------------------------------------
-RUN pip install --no-cache-dir \
-    mkdocs \
-    pyserial
+COPY requirements.txt .
```

Comment:
don't need this COPY line idt

Reply:
base directory of Embedded-Sharepoint will be mounted when the devcontainer is started up

Reply:
Seems to break without COPY. If we continue to add more files in this manner we could think about doing a mass copy into the image instead of line by line add for each required file.